src/board_controller/build.cmake

```diff
@@ -229,12 +251,8 @@ if (ANDROID)
 endif (ANDROID)
 
 if (UNIX AND NOT ANDROID)
-    target_link_libraries (${BOARD_CONTROLLER_NAME} PRIVATE pthread dl)
+    target_link_libraries (${BOARD_CONTROLLER_NAME} PRIVATE pthread dl m c)
```

Comment:
what is mc and how its related to pieeg?

Reply:
Removed unnecessary 'm' and 'c' libraries from target_link_libraries.